### PR TITLE
Added 03001-walz.js

### PR DESCRIPTION
**Pull request by izaakwalz**
Change hng-id to HNG-ID I also added my email address

### CODE REVIEW

**Comment by izaakwalz** [comment]
Done😉

File: scripts/03001-walz.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const HNG_ID = '03001';
+const full_name = 'Izaak Chukwuma';
+const language = 'JavaScript';
+
+console.log(
+  `Hello World, this is ${full_name} with HNGi7 ID hng-${HNG_ID} using ${language} for stage 2 task `
```
Comment (ifatoki) [inline scripts/03001-walz.js]: Please use upper case for the HNG in your HNG-ID

**Review by ifatoki — CHANGES_REQUESTED** [review]

File: scripts/03001-walz.js (hunk 1 above)
Comment (izaakwalz) [reply scripts/03001-walz.js]: Change hng-id to HNG-ID I also added my email address

Comment (lindaokorie27) [reply scripts/03001-walz.js]: The change he meant is like so "HNG-12345" within the string

**Review by melissaugrai — APPROVED** [review]

**Review by melissaugrai — APPROVED** [review]

**Review by melissaugrai — APPROVED** [review]

**Review by ifatoki — APPROVED** [review]

**Pull request merged**